static/app/utils/discover/fields.tsx

```diff
@@ -964,7 +969,7 @@ export function generateFieldAsString(value: QueryFieldValue): string {
 }
 
 export function explodeField(field: Field): Column {
```

Comment:
I dont think functions who's sole responsibility is to call one other function with proper params should exist. Seems like an overkill when you could just `explodeFieldString(field.field, field.alias)` everywhere